docs/LwM2M_Client/Nordic/Runtime_certificate.md

```diff
@@ -0,0 +1,105 @@
+# Runtime Certificate and Private Key Configuration
+
+## Introduction
+Anjay library supports secure connection between device and server. Encryption can be achieved with the use of Pre-Shared Key as it was described in [nRF9160](nRF9160.md) and [Thingy:91](Thingy91.md) pages or with certificate-based cryptography.
+
+In **Certificate Mode** an asymmetrical algorithm is used to authenticate the connection endpoints and initialize payload encryption. Appropriate certificates need to be generated for both the LwM2M Client and the LwM2M Server. Public certificates of both parties are mutually available, and each party also has access to its corresponding private key.
+
+You can read more about secure communication on [Anjay's documentation](https://avsystem.github.io/Anjay-doc/BasicClient/BC-Security.html) page.
+
+## Prerequisites
+* The nRF9160 DK board with a USB cable.
+* Cloned [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client#getting-started) repository
+
+!!! Note
+    Go through the README's `Getting Started` part.
+
+* Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
```

Comment:
Is that explicitly needed? I think this is installed while doing 
```
   west config manifest.file west-nrf.yml
    west update
 ```

Reply:
Let's leave this as it was. It's better to have this than fighting with possible problems later